spec/board_transformer_spec.rb

```diff
@@ -0,0 +1,82 @@
+require_relative '../lib/board_transformer'
+
+describe 'Board Transformer' do
+    include BoardTransformer
+    context 'Horizontal rows' do
+        it 'converts board data to horizontal rows' do
+            board = Board.new(size: 2)
+            board.update(position: 0, token: 1)
+            board.update(position: 1, token: 2)
+            board.update(position: 2, token: 3)
+            board.update(position: 3, token: 4)
+            expect(horizontal_rows(board)).to eql [
+                [1, 2],
+                [3, 4],
+            ]
+        end
+    end
+
+    context 'Vertical rows' do
+        it 'converts 2x2 board data to vertical rows' do
+            board = Board.new(size: 2)
+            board.update(position: 0, token: 1)
+            board.update(position: 1, token: 2)
+            board.update(position: 2, token: 3)
+            board.update(position: 3, token: 4)
+            expect(vertical_rows(board)).to eql [
+                [1, 3],
+                [2, 4],
+            ]
+        end
+
+        it 'converts 3x3 board data to vertical rows' do
+            board = Board.new(size: 3)
+            board.update(position: 0, token: 1)
```

Comment:
Is there a way to set up the boards for these tests without calling the update function a bunch of times? It would make the tests clearer